frontend/src/CreateShowingForm.js

```diff
@@ -170,8 +172,11 @@ class NewShowing extends React.Component {
           />
           <Field text="Datum:">
             <DatePicker
-              selected={showing.date}
-              onChange={v => this.setShowingDate(v)}
+              numberOfMonths={1}
```

Comment:
Number of months?

Reply:
hur många månader pickern ska visa, per sida